app/views/quotations/_quotations_table.html.erb

```diff
@@ -0,0 +1,50 @@
+<div class="content">
+        <div class="material-datatables">
+          <table id="quotation_table" class="table table-no-bordered table-hover" >
+            <thead>
+              <tr>
+                <th>Código</th>
+                <th>Código de Solicitud</th>
+                <th>Laboratorio</th>
+                <th>Fecha de Recepción</th>
+                <th>Estado</th>
+                <th>Acciones</th>
+              </tr>
+            </thead>
+            <tbody>
+              <% quotation.each do |q| %>
+                <tr>
+                  <td><%= q.code %></td>
+                  <td><%= q.code_Sol %></td>
+                  <td><%= q.laboratory %></td>
+                  <td><%= q.date %></td>
+                  <td><%= q.state %></td>
+                  <td>
+                    <%= link_to '<i class="fa fa-pencil" aria-hidden="true"></i>'.html_safe, edit_quotation_path(quotation), class: "btn btn-success btn-sm", data: { "toggle" => "tooltip" }, title: "Editar" %>
```

Comment:
You can use the <% link_to do %> notation to wrap some html with an anchor tag. 

See this example https://stackoverflow.com/questions/1086360/how-do-i-wrap-link-to-around-some-html-ruby-code